src/test/java/de/tum/in/www1/artemis/util/ModelFactory.java

```diff
@@ -997,62 +998,38 @@ public static LockRepositoryPolicy generateLockRepositoryPolicy(int submissionLi
     /**
      * Creates a dummy DTO used by Jenkins, which notifies about new programming exercise results.
      *
-     * @param repoName name of the repository
-     * @param successfulTestNames names of successful tests
-     * @param failedTestNames names of failed tests
-     * @param programmingLanguage programming language to use
+     * @param repoName                    name of the repository
```

Comment:
```suggestion
     * @param fullName                    ???
     * @param repoName                    name of the repository
```
The `@param` is missing, but I‘m not entirely sure what that parameter is.